docs/deployment-and-operations/packaging.md

```diff
@@ -38,7 +38,7 @@ The provided base image allows teams to package their applications with all the
 Below is an example Dockerfile for building a Stateful Functions image with both an [embedded module]({{ site.baseurl }}/sdk/modules.html#embedded-module) and a [remote module]({{ site.baseurl }}/sdk/modules.html#remote-module) for an application called ``statefun-example``.
 
 {% highlight dockerfile %}
-FROM flink-statefun:{{ site.version }}
+FROM ververica/flink-statefun:{{ site.version }}
```

Comment:
I'd prefer not to change the base image mentioned here, as the `ververica/` hosted one is considered a temporary solution for the time being. We do mention the Ververica image as a side note below, so this should be fine?

Reply:
I agree that `flink-statefun` without the `ververica` repository is preferrable, but I think we haven't published the image as an official library image yet (at least I can't `docker pull flink-statefun:2.1.0`), right?

My reasoning for the change was that with the `ververica` repository prefix the Dockerfile snippets work without requiring users to manually build the statefun image locally. What do you think?

(I'm happy to revert this change. I actually kept it as a separate commit, because I was expecting that this might have been on purpose.)

Reply:
I agree with @tzulitai in principle, but it doesn't look like there is any timeline on getting the official docker images out. I think we add the ververica registry for now, we can always retroactivly remove it from old versions of the docs after they are published.

Reply:
So, :shipit:?

Reply:
Maybe that should be brought up to a vote/discussion in the mailing list? what do you think @uce can you raise that issue?
My personal opinion here, that currently it is not a nice experience for anyone how would like to try the project and I don't see any progress in https://github.com/docker-library/official-images/pull/7749

Reply:
I had pinged Igal offline to start the discussion, because he was involved in the migration away from Ververica.

Since this is controversial, I will revert the commit and hope that we can merge this.

Reply:
Reverted.